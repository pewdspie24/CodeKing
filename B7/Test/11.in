64
10 3694515397
105369283 678738918
976569582 22361658
480796576 179576321
739683458 995160192
933609089 290385777
471528482 352689024
420908816 377871486
633093731 431877055
806457023 314870284
518667466 995677168
10 2213998205
111589732 786968815
135636815 143356044
178867246 921063195
538198940 752132064
907983737 817434070
817125197 240725571
843376557 997305676
115295998 380208277
705650090 285097557
413399636 658773374
8 992562696
142518056 21761215
738288259 335112217
631311353 699197462
190641001 344821433
101999681 135878600
575696309 130702616
375615250 406204184
36692239 437901390
2 80170378
376630774 203586279
341920674 581886282
10 3896722066
897725552 238505063
49780830 835700158
3055235 929704272
512652171 801990254
308296998 195380984
319397186 770417996
110835332 721840318
567524912 843960629
836688129 824820327
762593138 532565205
1 2588548298
778152731 427647907
1 727147328
150219286 922722968
7 4297477312
973526947 181426170
599773246 892542806
666332617 343264675
522428912 277823258
845302959 942478509
330052020 426763417
6954960 923398169
4 2442441463
727833050 735339545
914034659 846431319
576700932 410826712
533270529 866156558
3 2072389775
280048967 889548424
310502887 102522389
304883248 571718952
2 915756298
435985705 263892985
278858552 208749625
10 2017233711
394542420 468597095
349827083 860860480
80268338 557585201
434437901 165648877
456416831 651617009
682276467 353196440
615293775 376463409
483115302 780941058
356579734 792027151
835151223 660198271
10 4169159083
521608680 515089953
61193961 830225449
418387853 582307736
505142331 336694659
450371755 585576908
471465720 734202348
229584678 267223947
824322886 731804532
412201155 212231300
327277834 414469700
3 3594114677
428481437 519014180
509065985 584299515
281961242 969609098
7 2082243496
275762686 945785659
624097720 52621352
55067461 461800032
788719740 615119593
858157714 379502250
306264491 109251960
146631701 892613165
8 443224966
368555071 317024330
28474446 303877001
167735301 523579151
437732970 413358971
287877652 473844222
652193772 205810043
149130771 342053903
665830171 21237314
2 1126690580
631383247 840396317
876146542 444705412
4 2035160358
108874647 30752826
602730687 30272132
977349264 559801889
911720245 33090109
4 1522480663
81190409 413047907
224993742 497353810
169898157 288280093
38706793 135148747
9 4064879901
287756341 838253775
921388917 44540834
734251077 502506116
509795001 558350360
391161554 717309829
948244383 573549507
261704416 153382144
642391760 236099022
192373603 161564886
5 2879696486
167808878 975243013
848784506 270036168
257033287 819753217
135405031 448403990
186379084 179048943
7 2921853988
582086631 444252304
612899474 416398044
142607988 3198039
609224957 684411108
162615307 695942198
53948350 707120939
785182263 424119192
2 1535246216
30431501 620601087
396955913 26829351
8 2246199127
100395256 116455601
160481648 855025845
231043894 426221507
657384000 550263169
48636676 608172278
248900729 964646305
74990851 834304202
349768954 948607988
7 2910216304
370313267 636411732
760897956 904483110
921040547 464006960
254885118 543803470
669485621 578867271
363476852 907267960
832223472 929656575
8 2081309456
774339243 953670980
276537581 167943981
964766633 100475693
137282908 379398963
416154934 278429961
597203847 521111815
89291792 887266246
325764052 933080868
7 3322161867
330142730 9046183
756173417 908685905
626477010 843364058
856436938 619633812
681614367 847570315
884637306 794650593
79419050 688662594
8 2123243928
105022049 649620165
463347226 701377187
139598760 200576658
24111912 988903671
324728388 702140492
533903142 880697968
559293104 20578531
695579886 47013495
5 1515557991
487258791 209352033
411414285 206594496
678713485 6048817
916233791 995691792
346724335 938723139
9 1446204293
417555654 117056958
372581987 789933183
966183189 804026334
980841848 764623466
28562205 688395577
706308911 3870096
381969044 29817540
183392332 667524993
249415714 587085561
6 1262526699
992385749 311909644
92898755 988768693
166606592 58010978
926495134 295648923
113438505 59864397
892982249 807761422
2 1505972028
571797688 276836375
852485320 716536094
5 126032242
181398321 891585244
407840663 500157385
611197195 915577454
759070692 616197513
285207059 753754456
10 2116898513
276808062 748972709
302642154 475787300
820806570 494357833
482416098 39370349
523522479 993590483
843063796 116515056
160536551 317775864
412188501 584556813
128422467 755129761
429586035 804547539
1 995467593
478539776 937816831
10 1939210968
222232329 315844557
793366403 180059375
970543051 309953931
46290859 804761587
894245870 618084981
389880405 200754791
653070739 9415810
292099857 494675222
282984449 409277927
697875522 483560288
8 1464147450
122829610 995254233
706931680 797173049
607976256 244760970
912492967 226914719
521088916 298177578
841986994 653867275
153170856 351854583
271554960 161443050
2 135624082
927615152 433302128
295708136 178716128
8 4059915720
235192566 504895023
445854435 227370318
319313625 910087955
363437686 205507388
804883956 122758091
412279146 460225345
416149966 5106099
767759796 223290035
1 2402723293
139500539 852057268
2 1031921134
325109492 131996121
709863905 237125528
5 3050301135
158709366 637286515
895978223 92479306
256039028 190557735
148954354 48392238
599829013 50563860
2 476326171
470735638 71658170
831298084 497840091
10 3853300268
284787692 936365919
590550737 225185461
334799169 662161981
271651202 271417894
444377341 408469255
52749341 192133377
362186030 831036095
857907633 946218175
280062770 195484730
835777417 188697570
10 3478995894
914342601 104192482
43147963 449075161
74103236 744314801
867245471 345206706
921632115 294766563
343417035 934329709
127577677 238522818
412243244 399820883
510091142 298972216
72404513 744083086
3 3997895185
746777868 630893134
965105919 893862893
794050139 209376381
3 1095407359
403328768 690293046
241252527 13294585
383591704 720284481
6 1391830341
62946986 27634169
840415117 506865930
275841008 376452876
61757684 247968139
525738004 173052842
625221196 610152691
9 2039089946
831995458 494518776
705621639 169495092
814040671 429524009
220708838 336444321
181022896 674689888
494888 627120768
182445336 819636332
220959406 5776031
957227152 14426432
10 2944620783
921670522 463727077
220255731 136682016
13645539 164011184
266418501 461119746
67612861 677694901
505547355 772935233
837464595 817376886
834357184 969392702
54589904 124601032
975185792 716098783
7 1556860042
416313389 633569265
334282484 323230773
496252365 938636416
2342080 604149882
765303238 40113673
231839206 312093946
599570153 156050777
2 3496585351
339135009 178673810
371132746 970813595
6 3808701319
733417345 574433186
359639889 291147092
687511339 856970926
623492384 322548236
518412286 836691541
189021433 289763380
3 1578622136
679434874 31606470
135978762 246656786
581656232 753204856
3 590641038
51279681 704694735
131086522 254526379
728556248 634477060
1 2109556270
117207153 692453164
9 3591999879
595705792 290587532
42755868 331144100
454002691 693435977
878575118 756307799
276480736 704127118
534942310 199210645
666402646 303147368
421484353 778000297
381431790 562711544
2 2902341712
927773290 285111474
981480015 815269150
8 1310103526
624831100 566258523
292800919 247161617
686088057 345439488
695462315 581684546
684576237 861561626
925864131 107312103
759817045 836554003
765951018 406197221
7 2099900228
229435506 731360141
187044747 404267496
111073420 351156209
637744094 288405076
502298598 446549155
899991014 443506438
564975545 817932837
8 900424200
627053722 629973307
562037808 990860286
151873021 895624895
525464111 215068256
150269463 346081468
539751754 422801130
803443747 121336013
219665901 60566953
1 2648759933
239030732 135592721
5 3488082505
935801589 252225665
751516930 976948275
886772047 141740194
111319888 523963876
292629305 771501618
3 4147899052
650107513 499874225
738149936 246646710
166229443 387203062
