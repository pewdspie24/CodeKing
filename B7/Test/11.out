4
4
2
1
6
7
MISSION FAILED!
6
11
10
4
5
18
MISSION FAILED!
4
1
2
3
MISSION FAILED!
5
5
7
5
16
6
3
9
4
2
2
2
4
1
3
MISSION FAILED!
3
2
1
6
MISSION FAILED!
2
4
1
6
5
7
5
3
3
6
3
21
12
3
1
MISSION FAILED!
9
4
2
4
2
25
5
8
